_data/simple-icons.json

```diff
@@ -8036,7 +8040,7 @@
         {
             "title": "Sega",
             "hex": "0089CF",
-            "source": "https://en.wikipedia.org/wiki/Sega#/media/File:Sega_logo.svg"
+            "source": "https://en.wikipedia.org/wiki/File:SEGA_logo.svg"
```

Comment:
Could use the wikimedia version:
https://commons.wikimedia.org/wiki/File:SEGA_logo.svg